Description: Insert defines or compilation will break after autotools adjusts headers
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2019-08-22
Forwarded: no

--- a/mxml.h
+++ b/mxml.h
@@ -25,7 +25,7 @@
 #  include <string.h>
 #  include <ctype.h>
 #  include <errno.h>
-
+#  include <stdarg.h>
 
 /*
  * Constants...
--- a/mxml-string.c
+++ b/mxml-string.c
@@ -14,6 +14,8 @@
  */
 
 #include "config.h"
+#include <stdio.h>
+#include <string.h>
 #include <stdlib.h>
 #include <stdarg.h>
 
--- a/mxml-private.h
+++ b/mxml-private.h
@@ -100,4 +100,8 @@
 extern _mxml_global_t	*_mxml_global(void);
 extern int		_mxml_entity_cb(const char *name);
 
+char *_mxml_vstrdupf(const char *format, va_list ap);
+char *_mxml_strdup(const char *s);
+char *_mxml_strdupf(const char *format, ...);
+
 #endif /* !_mxml_private_h_ */
